docu/files/SkeletorInBuilderCLI.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Design a Builder UI with Skeletor'
+service: 'Builder SDK'
+area: 'Core'
+type: Tutorial
+order: -
+---
+
+The Builder CLI features the Skeletor tool with which you can easily create views just by dragging and dropping UI components. <br><br>
+
+<h3> How to use the Skeletor</h3>
```

Comment:
The Style and Standards CoP agreed not to use "How to" titles in the Dev Portal. Titles for tutorials should be in the imperative tense, for example: "Use the Skeletor tool". However, I'd recommend writing a title that clearly expresses what you're using the tool to do. For example: "Configure Skeletor for use with the Builder CLI" or whatever makes sense here.